index.html

```diff
@@ -2,69 +2,120 @@
 <html>
 <head>
 	<title>HTML forms</title>
+	<link href="styles.css" rel="stylesheet" type="text/css">
+	<link href="https://fonts.googleapis.com/css?family=Nunito&display=swap" rel="stylesheet">
+	<meta charset="utf-8">
 </head>
 <body>
 	<main>
-		<div>
-			<div>
-				<div>
-					<div></div>
-					<a href=""></a>
+		<div class="sign-up" style="height: 60px;">
+			<div class="s-u-section">
+				<div class="s-u-left">
+					<div class="accnt-msg">Already have an Intuit account?</div>
+					<div class="accnt-msg">I forgot my User ID or Password</div>
 				</div>
-				<div>
-					<button>
-						<span>
+				<div class="s-u-right">
+					<button class="btn">
+						<span class="lock-img"></span> Sign In
 							
-						</span>
+						
 					</button>
 				</div>
 			</div>
 		</div>
 
-		<header>
-			<div>
-				<a href=""></a>
+		<header class="header-style">
+			<div class="logo-container">
+				<a href="" class="title-link">
+					
+				</a>
+				<img src="img/logoIntuit.png" class="logo-intuit">
```

Comment:
`SUGGESTION`

Consider adding an `alt` attribute to the `<img>` element, this is a very important accessibility attribute.
`Examples of scenarios where users benefit from text alternatives for images` are listed below and could also be found [here](https://www.w3.org/TR/html51/semantics-embedded-content.html#alt-text)

- They have a very slow connection and are browsing with images disabled.
- They have a vision impairment and use text to speech software.
- They have a cognitive impairment and use text to speech software.
- They are using a text-only browser.
- They are listening to the page being read out by a voice Web browser.
- They have images disabled to save on download costs.
- They have problems loading images or the source of an image is wrong.


##### SOURCE: [W3C Recommendation (Embedded Content)](https://www.w3.org/TR/html51/semantics-embedded-content.html#alt-text)